disabled-packages/xfce4-goodies/build.sh

```diff
@@ -0,0 +1,14 @@
+# Error in the termux_step_create_debian_package.sh:49
+# script
+TERMUX_PKG_HOMEPAGE=https://goodies.xfce.org/
+TERMUX_PKG_DESCRIPTION="this meta-package will install various utilities and programs for the XFCE desktop environment (incomplete!)"
+TERMUX_PKG_LICENSE="Public Domain"
+TERMUX_PKG_MAINTAINER="Yisus7u7 <jesuspixel5@gmail.com>"
+TERMUX_PKG_VERSION=4.14.0
+TERMUX_PKG_SKIP_SRC_EXTRACT=true
+TERMUX_PKG_DEPENDS="xfce4-terminal, xfce4-appfinder, xfce4-calculator-plugin, xfce4-clipman-plugin, xfce4-datetime-plugin, xfce4-dict, xfce4-eyes-plugin, xfce4-mailwatch-plugin, xfce4-netload-plugin, xfce4-notes-plugin, xfce4-notifyd, xfce4-panel-profiles, xfce4-taskmanager, xfce4-timer-plugin, xfce4-wavelan-plugin, xfce4-whiskermenu-plugin, thunar-archive-plugin, parole, xarchiver, leafpad"
+TERMUX_PKG_CONFLICTS="xfce-goodies"
+TERMUX_PKG_REPLACES="xfce-goodies"
+TERMUX_PKG_PROVIDES="xfce4-plugins"
```

Comment:
Package conflicts with itself, replaces and provides itself? Are you sure that these fields should be set for the newly added package?